client/src/Router.js

```diff
@@ -1,20 +1,38 @@
-import React from "react";
+import React, { Component } from "react";
 import { Switch, Route, BrowserRouter } from "react-router-dom";
-import { AddFlight, Login, UpdateFlight } from "./pages";
+import { AddFlight, Login, UpdateFlight, Flights } from "./pages/admin-pages";
 
-const Router = () => (
-  <BrowserRouter>
-    <Switch>
-      <Route
-        exact
-        path="/"
-        render={() => <h3>Home page under construction</h3>}
-      />
-      <Route path="/admin" render={() => <Login />} />
-      <Route path="/addflight" component={AddFlight} />
-      <Route path="/updateflight" component={UpdateFlight} />
-    </Switch>
-  </BrowserRouter>
-);
+class Router extends Component {
+  state = {
+    flightNo: ""
+  };
+  handleFlightNumber = (e, data) => {
+    sessionStorage.setItem("flightNo", data);
+    this.setState({ flightNo: data });
+    window.location = "/admin/updateflight";
```

Comment:
Another way you could handle passing the flightNo to the UpdateFlight is by using a URL parameter with react-router.
Your route for UpdateFlight would include the flightNo in the path,
`<Route path="/admin/updateflight/:flightNo" ...`
Then inside UpdateFlight, you would be able to get flightNo out of this.props.match.params:
`const { flightNo } = this.props.match.params`
And then you wouldn't need the handleFlightNumber callback or sessionStorage to connect it all together, you could just link to that url.  In the Flights component, you could wrap the `<tr>` with a 
```
<Link to=`/admin/flights/${flightNo}`>
```

And then as an added bonus, you'd have deep links for updating flights!